doc/source/file_formats.rst

```diff
@@ -46,9 +46,48 @@ experimental support for NeuroLucida .asc files.
 SWC
 ---
 
-The SWC format represents a neuron as a tree of 3D points with additional information.
+The SWC format represents a neuron as a set of trees that are connected to a soma.
+The neuronal morphology is encoded as a rooted tree of 3D points and the corresponding radii.
 More information can be found `here <http://research.mssm.edu/cnic/swc.html>`_.
 
+The points of type "1" that represent the soma, have to be in the beginning of the file. 
+The soma can be represented in one of the following formats: 
+
+TypeA: One point soma
+^^^^^^^^^^^^^^^^^^^^^^
+
+The soma is represented by its center and a radius that corresponds to the sphere
+that preserves the surface area of the soma. The center of the soma has as parent ID -1 
+and all the initial points of the neuronal trees are connected to the soma. 
+
+TypeB: Three point soma
```

Comment:
Here, the point with parent ID -1 is a soma point and has two children. So, it is a soma point with a bifurcation into soma points. That is not allowed anymore in NeuroM but it is one of the SWC "standard" configurations. So, this kind of bifurcation **should** be allowed, but isn't, since #475.